src/main/java/befaster/runner/ClientRunner.java

```diff
@@ -158,7 +158,7 @@ private void executeServerActionFromUserInput(String[] args) {
     }
 
     private String getUserInput(String[] args) throws IOException {
-        return args.length > 0 ? args[0] : readInputFromConsole();
+        return args.length > 0 && !args[0].isEmpty() ? args[0] : readInputFromConsole();
```

Comment:
How does the args[0] ends up being empty?

Reply:
I was finding if you run `./gradlew run`, you would get an error:
![screen shot 2017-11-28 at 09 22 05](https://user-images.githubusercontent.com/5319573/33311726-a691c054-d41d-11e7-831b-de5287ad6d75.png)